### PR TITLE
Fix bug

### PR DESCRIPTION
**Pull request by grabbou**
Found it just right now during some `setTimeout` investigations. Makes it impossible to close menu.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -338,6 +339,7 @@ SideMenu.defaultProps = {
     );
   },
   defaultOpen: false,
+  isOpen: false
```
Comment (grabbou) [inline index.js]: is that going to play well with `defaultOpen: true` ?

**Comment by Kureev** [comment]
Can you also deprecate `defaultOpen`? It doesn't make any sense now

**Comment by Kureev** [comment]
Can you rebase you PR by latest master?

**Comment by grabbou** [comment]
Thanks!

**Pull request merged**